daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -873,8 +883,6 @@ object Main {
 
               val parseResult = eitherOutputterOrHandler match {
                 case Right(saxContentHandler) =>
-                  // reset in case we are streaming
-                  saxContentHandler.reset()
```

Comment:
Should we error or warning if streaming is enabled? The result of streamed EXI seems useless since we throwing NUL bytes as separators. And it's going to be impossible to unparse stream since EXI data likely has NUL bytes in it that we use for a separator In fact, we use a java.util.Scanner for finding the NUL delimiter, which is text based. So even if there were no NUL bytes in the input aside from our delimiter, I'm imagine things still wouldn't work. We'll need a completely different mechanism for supporting streaming EXI.

Reply:
We should error here until we know what we're doing with trying to use EXI for streaming. I have no idea if EXI files concatenated can be processed in any sensible way, are self-length aware, etc. XML is not sensible this way. It really wants to read until "end of data" before issuing an endDocument() SAX call/event. Some experiments are really needed here to see whether a sensible stream of SAX events can come from concatenated EXI data, or not. We may find that EXI streaming requires trickery like putting a length header on each chunk.

Reply:
(Sorry that comment from "mbeckerle-yahoo" which is a test account. I will login now with my regular userid. )

Reply:
Ok. Back signed in with proper id. 
So my comment is "yeah,.... what that yahoo up there said"

Reply:
Skimming through the EXI spec, it says:

> An EXI body can represent an EXI document with a single root element or an EXI fragment with zero or more root elements

Maybe when when streaming we use the "fragment" mode and each "infoset" is treated as a fragment. Without reading the spec more throughly though, it's not clear to me exactly what a fragment represents is and if it is streamable. But might be another option and something to look into if concatenated EXI's doesn't work.

Note that I don't think we need to add this support in this PR. An error as Mike suggests seams reasonable for now.

Reply:
An error message has been added if streaming mode is set with EXI infosets